src/overlays/actors/ovl_En_Crow/z_en_crow.c

```diff
@@ -21,41 +30,447 @@ const ActorInit En_Crow_InitVars = {
+void EnCrow_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnCrow* this = THIS;
+    f32 pad;
+    f32 height;
+    f32 scale;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E03B4.s")
+    func_809E1174(this, globalCtx);
+    this->actionFunc(this, globalCtx);
+    scale = this->actor.scale.x * 100.0f;
+    this->actor.posRot.rot.y = this->actor.shape.rot.y;
+    this->actor.posRot.rot.x = -this->actor.shape.rot.x;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E06D4.s")
+    if (this->actionFunc != func_809E10A8) {
+        if (this->actor.colChkInfo.health != 0) {
+            height = 20.0f * scale;
+            func_8002D97C(&this->actor);
+        } else {
+            height = 0.0f;
+            Actor_MoveForward(&this->actor);
+        }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E06E8.s")
+        func_8002E4B4(globalCtx, &this->actor, 12.0f * scale, 25.0f * scale, 50.0f * scale, 7);
+    } else {
+        height = 0.0f;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0770.s")
+    this->collider.list->dim.worldSphere.center.x = this->actor.posRot.pos.x;
+    this->collider.list->dim.worldSphere.center.y = this->actor.posRot.pos.y + height;
+    this->collider.list->dim.worldSphere.center.z = this->actor.posRot.pos.z;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0864.s")
+    if (this->actionFunc == func_809E0C8C) {
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0C8C.s")
+    if (this->collider.base.acFlags & 1) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0E2C.s")
+    if (this->actionFunc != func_809E10A8) {
+        CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0F54.s")
+    Actor_SetHeight(&this->actor, height);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E1004.s")
+    if (this->actor.colChkInfo.health != 0 && func_800A56C8(&this->skelAnime, 3.0f)) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_KAICHO_FLUTTER);
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E10A8.s")
+s32 EnCrow_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                            Actor* thisx) {
+    EnCrow* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E1174.s")
+    if (this->actor.colChkInfo.health != 0) {
+        if (limbIndex == 7) {
+            rot->y += 3072.0f * sinf(this->skelAnime.animCurrentFrame * 0.7853982f);
```

Comment:
```suggestion
            rot->y += 0xC00 * sinf(this->skelAnime.animCurrentFrame * (M_PI / 4));
```
Assuming the above matches, of course.